### PR TITLE
woboq_codebrowser 2.1

### PR DESCRIPTION
**Pull request by guruz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/woboq_codebrowser.rb [hunk 1]
```diff
@@ -1,14 +1,8 @@
 class WoboqCodebrowser < Formula
   desc "Generate HTML from C++ Code"
   homepage "https://code.woboq.org/"
-  url "https://github.com/woboq/woboq_codebrowser/archive/2.0.1.tar.gz"
-  sha256 "f2ec67bb49415986a4bbef515024009433756c83e14fa84f5acc40c83eec19e6"
-
-  bottle do
```
Comment (ilovezfs) [inline Formula/woboq_codebrowser.rb]: please do not alter the bottle block.

Comment (guruz) [inline Formula/woboq_codebrowser.rb]: i thought the bottle is for the old version and the bot will re-put a new block? what now, shall i re-add it?

Comment (guruz) [inline Formula/woboq_codebrowser.rb]: (i did not alter the block, i removed it)

Comment (ilovezfs) [inline Formula/woboq_codebrowser.rb]: removal = altering

>i thought the bottle is for the old version and the bot will re-put a new block? what now, shall i re-add it?

Yes, re-add it, please. Take a look at the commit history to see how this works.

Comment (guruz) [inline Formula/woboq_codebrowser.rb]: i re-pushed

**Comment by ilovezfs** [comment]
Thanks for the PR @guruz! Shipped!

**Comment by guruz** [comment]
grrrrreat!

**Pull request closed without merging**